src/blocks/tipset.rs

```diff
@@ -282,6 +282,15 @@ impl Tipset {
         }
         broken
     }
+    /// Returns an iterator of all tipsets
```

Comment:
Of all ancestor tipsets

Reply:
since it includes itself also,  @lemmih  suggested we keep it to `Returns an iterator of all tipsets`